src/askscript/__tests__/03-lambda/lambda-04.ask

```diff
@@ -0,0 +1,3 @@
+ask() {
+  call(get('toString'), 2)
```

Comment:
This test is missing an expected output, hence it's not executed at all.
When I paste 'call(get('toString'), 2)' to CLI, I get:
```
Uncaught Error: Cannot call null
```

Reply:
Here's a test that says otherwise: https://github.com/xFAANG/askql/pull/197